JS/app.js

```diff
@@ -25,19 +26,49 @@ photoArray.push(new Photo('img/kitten12.jpg'));
 photoArray.push(new Photo('img/kitten13.jpg'));
 photoArray.push(new Photo('img/kitten14.jpg'));
 
-//etc
+var leftVotes;
+var rightVotes;
 
-var Tracker = function() {
-this.leftPhoto = this.getRandomInt();
-this.rightPhoto = this.getRandomInt();
+var ctx = document.getElementById("myChart").getContext("2d");
+console.dir(Chart)
```

Comment:
Make sure you remove these console.log / debug statements from your production code.